### PR TITLE
[BACKLOG-31896] MARKETO - CSFR - /api/repo/files/import

### PR DESCRIPTION
**Pull request by dcleao**
@pentaho/millenniumfalcon please review.

### CODE REVIEW

File: extensions/src/main/java/org/pentaho/platform/plugin/services/importexport/CommandLineProcessor.java [hunk 1]
```diff
@@ -20,12 +20,9 @@
 
 package org.pentaho.platform.plugin.services.importexport;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
+import java.io.*;
```
Comment (davidmsantos90) [inline extensions/src/main/java/org/pentaho/platform/plugin/services/importexport/CommandLineProcessor.java]: avoid .* imports

File: extensions/src/test/java/org/pentaho/platform/plugin/services/importexport/CommandLineProcessorTest.java [hunk 2]
```diff
@@ -102,8 +102,11 @@ public void before() {
   }
 
   private void testRequestType( String keyKey, String nameKey, RequestType rt ) throws ParseException {
-    CommandLineProcessor clp = createCommandLineProcessor( keyKey, nameKey );
-    assertEquals( rt, clp.getRequestType() );
+    try ( CommandLineProcessor clp = createCommandLineProcessor( keyKey, nameKey ) ) {
+      assertEquals( rt, clp.getRequestType() );
+    } catch (IOException e) {
```
Comment (davidmsantos90) [inline extensions/src/test/java/org/pentaho/platform/plugin/services/importexport/CommandLineProcessorTest.java]: missing whitespaces

**Comment by dcleao** [comment]
@davidmsantos90 done!

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 21m 51s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl extensions
```
---
:ok_hand: All tests passed!
 
Tests run: **1470**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/platform/job/PR-4584/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**